src/services/ImageRepository.cpp

```diff
@@ -262,7 +258,7 @@ static HRESULT ConvertBitmapSource(_In_ RECT rcDest, _In_ IWICBitmapSource* pOri
     return hr;
 }
 
-static HRESULT CreateDIBFromBitmapSource(_In_ IWICBitmapSource *pToRenderBitmapSource, _Out_ HBITMAP& hBitmapInOut)
+static HRESULT CreateDIBFromBitmapSource(_In_ IWICBitmapSource *pToRenderBitmapSource, _Inout_ HBITMAP& hBitmapInOut)
```

Comment:
This is only called in a situation where hBitmapInOut is truly just an out parameter. The name should be fixed, and the "DeleteBitmap" check on line 299 is unnecessary.

I'm fine with leaving this for now.